django/university/routes/student/schedule/StudentScheduleView.py

```diff
@@ -15,7 +15,7 @@ def get(self, request, format=None):
 
         try:
             response = requests.get(get_student_schedule_url(
-                request.user.username, 
+                202208817, #request.session["username"],
```

Comment:
Could you change this back to `request.user.username`?

Reply:
Done